src/index.js

```diff
@@ -13,8 +13,9 @@ import ComponentStory from './routes/story-page/componet'
 import ButtonStory from './routes/story-page/button'
 import QuickTransfer from './components/quickTransfer/QuickTransfer'
 import NotificationStory from './routes/notification-page/index'
-import Payment from './routes/payment/payment'
+import Activites from './routes/recent-activite/recent-activite'
```

Comment:
Just a typo. Please, also make sure to update the component name, folder name, link, and path.

```suggestion
import Activities from './routes/recent-activity/recent-activity
```

Reply:
- fixed the typos "Activites" -> "Activities" everywhere
- add blanc lines between selectors in css file
- completed the main post with screenshot and link to issue